Library/Formula/ccnet.rb

```diff
@@ -0,0 +1,61 @@
+class Ccnet < Formula
+  homepage "http://www.seafile.com/"
+  url "https://github.com/haiwen/ccnet/archive/v4.2.3.tar.gz"
+  sha256 "7da578ed4b04b99673c711cf22a9dc535e31c6f323e5701348211129be48c92d"
+
+  head "https://github.com/haiwen/ccnet.git"
+
+  # [FIX] homebrew autotools path
```

Comment:
Can you please submit this patch to the upstream developers of this project (if you haven't already) and add a link to the upstream pull-request and explanation of why the patch is needed in a comment in the formula file. Thanks!